modules/shared/src/main/scala/org/tessellation/kryo/KryoSerializer.scala

```diff
@@ -13,7 +13,7 @@ import com.twitter.chill._
 
 trait KryoSerializer[F[_]] {
 
-  def serialize(anyRef: AnyRef): Either[Throwable, Array[Byte]]
+  def serialize[A](anyRef: A): Either[Throwable, Array[Byte]]
```

Comment:
Not an `anyRef` anymore, maybe we should call it `data` or simply `a` 🤔

Reply:
Do you remember why it was an `AnyRef` originally? Wasn't it because of some Kryo's requirement/limitation? Can we change it then?

Reply:
I don't remember, probably it was a leftover or sth. Everything serializes correctly, even the primitives, so it shouldn't be an issue